features/admin.actions.v1/pages/action-configuration-page.tsx

```diff
@@ -132,23 +148,51 @@ const ActionConfigurationPage: FunctionComponent<ActionConfigurationPageInterfac
         if (actionsFetchRequestError.response?.data?.description) {
             dispatch(
                 addAlert<AlertInterface>({
-                    description: t("actions:notification.error.fetch.description",
+                    description: t("actions:notification.error.fetchByType.description",
                         { description: actionsFetchRequestError.response.data.description }),
                     level: AlertLevels.ERROR,
-                    message: t("actions:notification.error.fetch.message")
+                    message: t("actions:notification.error.fetchByType.message")
                 })
             );
         } else {
             dispatch(
                 addAlert<AlertInterface>({
-                    description: t("actions:notification.genericError.fetch.description"),
+                    description: t("actions:notification.genericError.fetchByType.description"),
                     level: AlertLevels.ERROR,
-                    message: t("actions:notification.genericError.fetch.message")
+                    message: t("actions:notification.genericError.fetchByType.message")
                 })
             );
         }
     }, [ isActionsLoading, actionsFetchRequestError ]);
 
+    /**
+     * The following useEffect is used to handle if any error occurs while fetching the Action by Id.
+     */
+    useEffect(() => {
+        if (isActionLoading || !actionFetchRequestError) {
+            return;
+        }
+
+        if (actionFetchRequestError.response?.data?.description) {
+            dispatch(
+                addAlert<AlertInterface>({
+                    description: t("actions:notification.error.fetchById.description",
+                        { description: actionFetchRequestError.response.data.description }),
```

Comment:
Are there error descriptions returned from the API always user friendly messages?

Reply:
Yes. All the server and client errors are properly handled in the backend.

Reply:
Ack